docker/common.docker

```diff
@@ -89,11 +89,14 @@ COPY --chown=www-data:www-data ./.gitattributes ./.gitattributes
 COPY --chown=www-data:www-data ./.git ./.git
 COPY --chown=www-data:www-data ./.gitignore ./.gitignore
 COPY --chown=www-data:www-data ./jest.config.js ./jest.config.js
+COPY --chown=www-data:www-data ./lang ./lang
 COPY --chown=www-data:www-data ./LICENSE ./LICENSE
 COPY --chown=www-data:www-data ./package.json ./package.json
 COPY --chown=www-data:www-data ./package-lock.json ./package-lock.json
 COPY --chown=www-data:www-data ./.php-cs-fixer.dist.php ./.php-cs-fixer.dist.php
 COPY --chown=www-data:www-data ./phpunit.xml ./phpunit.xml
+COPY --chown=www-data:www-data ./phpstan-baseline.neon ./phpstan-baseline.neon
+COPY --chown=www-data:www-data ./phpstan.neon  ./phpstan.neon
```

Comment:
Would it be better to only copy these files if it's a development build?

As an aside, is there a way to copy these files over without having to manually create entries like this for every file?  I'm also a little confused about how my initial PHPStan PR was able to see these files in the first place.

Reply:
As to the last point: The CircleCI script seems to `checkout` the branch again just before the tests are run:
 https://app.circleci.com/pipelines/github/Kitware/CDash/1033/workflows/c49baff4-29c9-4b0f-8ac0-0d17e8be645b/jobs/4534?invite=true#step-104-30.  So, even if the files are missed in the copy to the Docker image, that line will ensure that they're found in the running tests.  

I'm not quite sure why we need to specify each file instead of passing the directory as a whole.  I'll see what we're leaving behind, if anything, and what we can do